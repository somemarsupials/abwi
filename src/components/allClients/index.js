export { default as ClientsPage } from './clientsList';
export { default as ClientsList } from './clientsPage';
