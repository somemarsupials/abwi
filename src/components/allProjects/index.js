export { default as List } from './list';
export { default as Page } from './page';
