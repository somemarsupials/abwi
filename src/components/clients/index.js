export { default as ClientPage } from './clientPage';
export { default as ClientDetail } from './clientDetail';
