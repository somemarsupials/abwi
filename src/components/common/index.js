export { default as TitleBar } from './titleBar';
export { default as ErrorBar } from './errorBar';
