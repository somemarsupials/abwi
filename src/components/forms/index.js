export { default as ProjectForm } from './projectForm';
