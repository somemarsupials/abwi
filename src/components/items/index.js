export { default as ItemList } from './itemList';
