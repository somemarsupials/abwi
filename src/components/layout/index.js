export { default as SubHeader } from './subHeader';
