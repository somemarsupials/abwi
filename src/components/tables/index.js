export { default as ClientTable } from './clientTable';
export { default as ProjectTable } from './projectTable';
export { default as ItemTable } from './itemTable';
