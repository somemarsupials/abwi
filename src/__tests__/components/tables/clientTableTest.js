import React from 'react';
import { shallow } from 'enzyme';

import { ClientTable } from '../../../components/tables';

describe('ClientTable', () => {
  let table;
  let clients;

  beforeEach(() => {
    clients = [{ name: 'a' }, { name: 'b' }]
  });

  describe('without clients', () => {
    beforeEach(() => {
      table = shallow(<ClientTable />);
    });

    it('renders with message', () => {
      expect(table).toMatchSnapshot();
    });
  });

  describe('with clients', () => {
    beforeEach(() => {
      table = shallow(<ClientTable clients={clients} />);
    });

    it('renders with clients', () => {
      expect(table).toMatchSnapshot();
    });
  });
});
