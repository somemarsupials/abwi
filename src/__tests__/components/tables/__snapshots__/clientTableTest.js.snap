// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientTable with clients renders with clients 1`] = `
<table
  className="table"
>
  <thead>
    <tr>
      <th>
        Name
      </th>
    </tr>
  </thead>
  <tbody>
    <tr
      key="0"
    >
      <td>
        <Link
          replace={false}
          to="/clients/undefined"
        >
          a
        </Link>
      </td>
    </tr>
    <tr
      key="1"
    >
      <td>
        <Link
          replace={false}
          to="/clients/undefined"
        >
          b
        </Link>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`ClientTable without clients renders with message 1`] = `
<Alert
  bsClass="alert"
  bsStyle="info"
  closeLabel="Close alert"
>
  No clients found
</Alert>
`;
